Kickstarter-iOS/Views/Controllers/CreatePasswordViewController.swift

```diff
@@ -48,13 +69,77 @@ final class CreatePasswordViewController: UITableViewController {
     self.tableView.registerCellClass(SettingsTextInputCell.self)
   }
 
+  override func viewDidAppear(_ animated: Bool) {
+    super.viewDidAppear(animated)
+
+    self.viewModel.inputs.viewDidAppear()
+  }
+
   // MARK: - Styles
 
   override func bindStyles() {
     super.bindStyles()
 
     _ = self.tableView
       |> settingsGroupedTableViewStyle
+      |> \.allowsSelection .~ true
```

Comment:
In order for the table to allow cell selection (that allows us changing first responder on tap and focusing on the text field even if user taps on the label)